check_pve.py

```diff
@@ -363,6 +364,18 @@ def checkStorage(self, name):
         url = self.getURL('nodes/{}/storage/{}/status'.format(self.options.node, name))
         self.checkAPIValue(url, 'Storage usage is')
 
+    def checkVersion(self):
+        url = self.getURL('version')
+        data = self.request(url)
+        if not data['version']:
+            self.checkResult = NagiosState.UNKNOWN
+            self.checkMessage = "Unable to determine pve version"
+        elif LooseVersion(self.options.min_version) >= LooseVersion(data['version']):
```

Comment:
Hey @nbuchwitz, thanks for merging! Unfortunately I made a mistake here. The comparsion operator should be `>` instead of `>=` (if you've got `min_version` you are also fine). I don't wanna open another PR just for this, so I'd appreciate a quickfix from you 👍

Reply:
I pushed the changes in 7305b8d